src/main/java/com/atlassian/bitbucket/jenkins/internal/model/BuildState.java

```diff
@@ -0,0 +1,56 @@
+package com.atlassian.bitbucket.jenkins.internal.model;
+
+import hudson.model.AbstractBuild;
+import hudson.model.Result;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+/**
+ * The state or result of a build
+ */
+public enum BuildState {
```

Comment:
I think we can tidy this up a bit, look at my suggestions in https://github.com/jenkinsci/atlassian-bitbucket-server-integration-plugin/pull/28